code-challenges.test.js

```diff
@@ -37,6 +85,28 @@ var colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saf
 
 // a) Create a test with expect statements for each of the variables provided.
 
+describe("when cubeTheSums is called", () => {
+    it("returns 99 when called with [2, 3, 4]", () => {
```

Comment:
test description should be generic and describe the behavior (not so much about the expected outcome and input